src/app/services/statusService.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+import config from '../config';
+
+const statusUrl = `${config.API_ENDPOINT}/api/status`;
+
```

Comment:
Can we remove the `service` prefix on the service filenames? Just `app/services/status.js` would be enough.